targets/unity-v2/source/ExampleTestProject/Assets/PlayFabEditorExtensions/Editor/PlayFabEditorExtensions.asmdef

```diff
@@ -1,8 +1,6 @@
-{
-    "name": "PlayFabTestingEditor",
-    "references": [
-        "PlayFab"
-    ],
+﻿{
+    "name": "PlayFabEditorExtensions",
+    "references": [],
```

Comment:
So I know this was definitely needed when I was implementing the changes, but I do see there are some functions being made public rather than internal. I'd like to manually test this changeset myself to ensure this change is needed (because if so I think a lot of this file potentially becomes unneeded as the default file generated only has 1 or 2 values)

Reply:
I think the "diff" here is misleading.
This is the new file copied as-is from Editor Extensions, and it should just appear as a pure add, not a diff.
The diff is a false comparison, and only seems to be here because the existing file is so close.
I just copied this straight from EdEx without any changes.

---
In reply to: [362102681](https://github.com/PlayFab/SDKGenerator/pull/638#discussion_r362102681) [](ancestors = 362102681)

Reply:
Ah sweet! I like that we now directly add the EditorExtension to SDKGenerator. There have been too many times testing the SDK where the editor extensions would be useful to have at the same time. 

Would we want to consider making these a git submodule in the UnitySDK repo? Or would we want to make this the final resting place of the editor extensions (and deprecate the separate public repo?)

---
In reply to: [362262535](https://github.com/PlayFab/SDKGenerator/pull/638#discussion_r362262535) [](ancestors = 362262535,362102681)